doc/install/contributing.rst

```diff
@@ -538,12 +538,31 @@ Describe your changes in the changelog
 --------------------------------------
 
 Include in your changeset a brief description of the change in the
-:doc:`changelog <whats_new>` (:file:`doc/whats_new.rst`; this can be skipped
-for very minor changes like correcting typos in the documentation). Note that
-there are sections of the changelog for each release, and separate subsections
-for bugfixes, new features, and changes to the public API. It is usually best
-to wait to add a line to the changelog until your PR is finalized, to avoid
-merge conflicts (since the changelog is updated with almost every PR).
+:ref:`changelog <whats_new>` (:file:`doc/changes/latest.inc`; this can be
+skipped for very minor changes like correcting typos in the documentation).
+The entry should follow the following patterns:
+
+.. code-block::
+
+    - Short description of the changes from one contributor (:gh:`0000` by `Contributor Name`_)
+    - Short description of the changes from several contributors (:gh:`0000` by `Contributor Name`_, `Second Contributor`_, and `Third Contributor`_)
+
+where ``0000`` must be replaced with the respective GitHub pull request (PR)
+number. Mind the Oxford comma in the case of multiple contributors.
+
+Sometimes, changes that shall appear as a single changelog entry are spread out
+across multiple PRs. In this case, simply name all relevant PRs, separated by
+commas:
+
+.. code-block::
+
+    - Short description of the changes from one contributor in multiple PRs (:gh:`0000`, :gh:`1111` by `Contributor Name`_)
+    - Short description of the changes from several contributors in multiple PRs (:gh:`0000`, :gh:`1111` by `Contributor Name`_, `Second Contributor`_, and `Third Contributor`_)
+
+Note that there are sections of the changelog for each release, and separate
+subsections for bugfixes, new features, and changes to the public API. It is
+usually best to wait to add a line to the changelog until your PR is finalized,
+to avoid merge conflicts (since the changelog is updated with almost every PR).
```

Comment:
Can you add a note about how new contributors should put their entry at the top of the appropriate section (with an example with the RST substitution business) and add their name to `names.inc`

Reply:
Done

Reply:
You didn't add the part about adding to the top of the section. @larsoner is this really how we handled it? I always put new entries at the bottom of the appropriate section. In any case, this should be mentioned here.

Reply:
At the top now?? Why's that?

Reply:
I think @larsoner meant at the bottom of the appropriate section.

Reply:
But we should make sure to mention this (whatever we decide, probably at the bottom).

Reply:
> You didn't add the part about adding to the top of the section. @larsoner is this really how we handled it? I always put new entries at the bottom of the appropriate section. In any case, this should be mentioned here.

Above I'm talking about *new contributors* -- we decided their entries should go at the top of the appropriate section. This should be added along with an example with the substitution syntax that allows for bolding of their name within a link as well.

Reply:
Ooooh!! Ok that does make sense indeed